app/services/variant_units/unit_prices.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module VariantUnits
+  class UnitPrices
+    def initialize(variant)
+      @variant = variant
+      @product = variant.product
+    end
+
+    def unit_price_denominator
+      return @variant.unit_value if @product&.variant_unit == "items"
+
+      case unit_price_unit
+      when "lb"
+        @variant.unit_value / 453.6
```

Comment:
it'd be great if we could give these magic numbers a name assigning them to a constant. I'm not at all familiar with `lb` and I have no idea what this is.

Reply:
I agree - I'm going to move it to a tech debt issue though if that's okay with you. It will touch a lot of files and I don't want it to bog down this PR. 

https://github.com/openfoodfoundation/openfoodnetwork/issues/7046